.drone.yml

```diff
@@ -11,7 +11,7 @@ platform:
 
 steps:
   - name: initialize
-    image: grafana/grafana-plugin-ci:1.2.1-alpine
+    image: grafana/grafana-plugin-ci:1.5.0-alpine
```

Comment:
This updates the node version. After updating some dependencies, one dependency required a more up-to-date version of node.